dotnet/samples/KernelSyntaxExamples/Example65_HandlebarsPlanner.cs

```diff
@@ -114,7 +115,7 @@ private static async Task RunSampleAsync(string goal, bool shouldPrintPrompt = f
         Console.WriteLine($"\nOriginal plan:\n{plan}");
 
         // Execute the plan
-        var result = plan.Invoke(kernel.CreateNewContext(), new Dictionary<string, object?>(), CancellationToken.None);
+        var result = plan.Invoke(new ContextVariables(), new Dictionary<string, object?>(), CancellationToken.None);
```

Comment:
Interesting, why do we need variables different from planVariables? Shouldn't they be the same?

Reply:
Eventually they will be the same. It's a way the handlebars planner supports complex types while the rest of SK does not..